plaso/lib/aul/activity.py

```diff
@@ -0,0 +1,243 @@
+# -*- coding: utf-8 -*-
+"""The Apple Unified Logging (AUL) Activity chunk parser."""
+
```

Comment:
seeing this is a parser maybe move this to plaso/parsers/shared same for other unified logging parser related files maybe plaso/parsers/shared/unified_logging ?